newsletter/hooks/usePaywall.js

```diff
@@ -42,7 +42,7 @@ sc.parentNode.insertBefore(js, sc); }(document, "script"));`)
       const handler = event => {
         // WARNING: THIS IS NOT DOCUMENTED!
         // Let's look at the data that we have
-        if (window.unlockProtocol) {
+        if (window.unlockProtocol && window.unlockProtocol.blockchainData) {
```

Comment:
Since the paywall is not completely initialized until the user clicks on the checkout button, the `blockchainData` function may not be defined.